view/src/components/produce.js

```diff
@@ -348,21 +348,21 @@ class Produce extends Component {
         .then(() => {
           // Page state
           this.setState({ open: false });
-          const message =
+          const alert =
```

Comment:
hm, `alert` isn't really right either tho, alert would be the entire message.
I really liked `action`, since it's the verb part of this phrase. (`Produce has been successfully + action` makes more sense to me)

Variable names are hard.